sound/soc/sof/core.c

```diff
@@ -455,10 +455,19 @@ EXPORT_SYMBOL(snd_sof_device_remove);
 int snd_sof_device_shutdown(struct device *dev)
```

Comment:
Oh, maybe remove the "FIXME:" in description. 

Hmm, yes, the trace should be stopped. In our current cases, this is not an issue as DMA trace is not enabled at all, but in general, trace could be involved.

The power-down notification I think we can omit as this is done by sof_suspend() (called via HDA shutdown path).

Reply:
it was intentional, I omitted the prefix :-)